packages/hash/playwright/tests/page-creation.spec.ts

```diff
@@ -105,6 +105,39 @@ test("user can create page", async ({ page }) => {
     blockRegionLocator.locator('[data-testid="block-handle"]'),
   ).toHaveCount(3);
 
+  await page.keyboard.press("Enter");
+  await sleep(100); // TODO: investigate flakiness in FF and Webkit
+
+  await expect(
+    blockRegionLocator.locator('[data-testid="block-handle"]'),
+  ).toHaveCount(4);
+
+  const blockChanger = blockRegionLocator
+    .locator('[data-testid="block-changer"]')
+    .nth(2);
+  await blockChanger.click();
+
+  await blockChanger
+    .locator('[placeholder="Load Block from URL..."]')
+    .fill("https://blockprotocol.org/blocks/@shinypb/emoji-trading-cards");
```

Comment:
I feel it might be a good idea to use internal blocks for tests. External blocks can be taken down without us knowing.

Reply:
I know, and am equally concerned by that – but all our internal blocks are loaded by default, so there isn't an alternative good option which wouldn't have taken much longer to do. We can and should look at replacing this.

Reply:
I see. I didn't consider that. I agree we should look at replacing it later.